lumibot/brokers/alpaca.py

```diff
@@ -107,13 +108,18 @@ class Alpaca(AlpacaData, Broker):
         forex=[],
     )
 
-    def __init__(self, config, max_workers=20, chunk_size=100, connect_stream=True):
+    def __init__(self, config, max_workers=20, chunk_size=100, connect_stream=True, data_source=None):
         # Calling init methods
-        AlpacaData.__init__(
-            self, config, max_workers=max_workers, chunk_size=chunk_size
-        )
-        Broker.__init__(self, name="alpaca", connect_stream=connect_stream)
         self.market = "NASDAQ"
+        self.api_key = ''
+        self.api_secret = ''
+        self.paper = False
```

Comment:
can we leave this as "is_paper"? It's a little more descriptive, so it's easy to tell that it's a boolean

Reply:
Done!